### PR TITLE
feat(auth): support phone_login [Experimental]

### PR DESCRIPTION
**Pull request by little-buddy**
router.meta.auth = true,
will do check login status.

This is the original version and I will continue to improve it in the next few days 😜

### CODE REVIEW

File: src/app/app.less [hunk 1]
```diff
@@ -29,3 +29,45 @@ li {
   background-color: #e2e2e2;
   border-radius: 2px;
 }
+
+.absoluteFill {
```
Comment (Linkontoask) [inline src/app/app.less]: Can it be changed inside the component because it is more intuitive.

File: src/pages/auth/component/AuthView.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+export const AuthView = defineComponent({
+  setup(props, { slots }) {
+    const $store = useStore()
+    const doClose = () => {
+      $store.commit('Auth/HIDE_VIEW')
+    }
+
+    onMounted(() => {
+      window.history.pushState(null, '123', location.href)
```
Comment (Linkontoask) [inline src/pages/auth/component/AuthView.tsx]: What is the reason for this?

**Review by Linkontoask — COMMENTED** [review]
Try to avoid too many file submissions, too many reviews are painful

File: src/pages/auth/component/AuthView.tsx (hunk 2 above)
Comment (little-buddy) [reply src/pages/auth/component/AuthView.tsx]: push empty route, than modal can be closed by router.back()

**Comment by little-buddy** [comment]
Wait for me, i will do better in this week

**Comment by Linkontoask** [comment]
I pulled your request to run and found the following errors, please correct them in time
```
These relative modules were not found:

* ../component/button in ./src/pages/auth/views/Signup.tsx, ./src/pages/auth/views/EmailLogin.tsx and 2 others
* ../component/link in ./src/pages/auth/views/Signup.tsx, ./src/pages/auth/views/PhoneLogin.tsx
```

The following points are not mandatory, but need to be unified.
+ Do not use the camel case method to name the folder, please use dashes to separate it.
+ Change the first letter of the file name to lowercase.

**Comment by Linkontoask** [comment]
I submitted some code, and there was a conflict. Iconfont/index.js is subject to your file. I added the following code snippet in router/index.js

https://github.com/Linkontoask/radishes/commit/bebec1d335cfff7eb2a1cc738aa308ab63366b76#diff-ed3eadd8a0891daf6ad6d6fabe767371981e5626c8ee467e694455fcbbb63710

**Comment by little-buddy** [comment]
ok, I will follow you. 🤦‍♂️

**Comment by Linkontoask** [comment]
Well, after I synced your submission, I found that it can't work normally in the windows environment. I guess it's caused by the file system. I strictly distinguish the case of the file path.  
Before merging this submission, I still need to do some things, such as testing whether the build is normal, I am writing an automatic test, and I will tell you here after it is completed, thank  ❤️

**Comment by Linkontoask** [comment]
There will be problems with the login interface under the ~~1920x1080 screen~~, because of the font. I recommend using padding and content padding.
image

**Comment by Linkontoask** [comment]
image

**Pull request merged**